README.md

```diff
@@ -95,8 +95,9 @@ After you're finished please remove all the comments and instructions!
 ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
-- This is Rails based project.
```

Comment:
- [ ] Please notice the section in your readme which start with `>` are directions/guidelines from microverse to help you understand what to write in that section and should not be part of your readme file. Please consider removing this and all the like contents from the readme file.